src/test/kotlin/org/rust/lang/core/resolve/RsResolveTest.kt

```diff
@@ -591,5 +591,73 @@ class RsResolveTest : RsResolveTestBase() {
                 //^ unresolved
         }
     """)
-}
 
+    fun testMethod() = checkByCode("""
```

Comment:
These tests belong to `TypeAwareResolveTest`. The split is done only to make the test file size manageable.